etc/notebooks/test/test_plotting_libs.ipynb

```diff
@@ -385,7 +386,7 @@
    },
    "outputs": [],
    "source": [
-    "show(p)"
+    "handle = show(p, notebook_handle=True)"
```

Comment:
I pulled down the code and ran as follows:

```
make clean
make dev-install kernel-gateway-image examples
make dev
```

Now I'm getting this error in the browser console when running this dashboard:

```
TypeError : show() got an unexpected keyword argument 'notebook_handle' 
 
TypeErrorTraceback (most recent call last)
<ipython-input-10-b378e9357e15> in <module>()
----> 1 handle = show(p, notebook_handle=True)

TypeError: show() got an unexpected keyword argument 'notebook_handle'
```